lib/versioning/cargo/index.ts

```diff
@@ -55,19 +55,27 @@ function npm2cargo(input: string): string {
 }
 
 const isLessThanRange = (version: string, range: string): boolean =>
-  npm.isLessThanRange(version, cargo2npm(range));
+  !!npm.isLessThanRange?.(version, cargo2npm(range));
```

Comment:
Are you sure we need this change?

Reply:
Yes, checked it using `yarn null-check`

- ? is necessary as per `VersioningApi` in `types.ts` and it makes the overall type (boolean|undefined) so,
- !! converts falsy values to boolean(false).